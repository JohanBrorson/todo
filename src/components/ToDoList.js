import React from 'react';
import ToDoItem from './ToDoItem';
import './ToDoList.css';

export default class ToDoList extends React.PureComponent {

  render() {
    const {
      items,
    } = this.props;

    return (
      <div className="center">
        <ul>
          {
            items && items.map((item, index) => (
              <ToDoItem
                key={index}
                id={index}
                todoItem={item}
                toggle={this.props.toggle}
                remove={this.props.remove}>
              </ToDoItem>
          ))}
        </ul>
      </div>
    );
  }
}
