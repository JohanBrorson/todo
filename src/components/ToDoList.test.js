import React from 'react';
import ReactDOM from 'react-dom';
import ToDoList from './ToDoList';
import { shallow } from 'enzyme'

describe('<ToDoList />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ToDoList />)
    wrapper.setProps("a", "b")
    expect(wrapper.getElement()).toMatchSnapshot();
  });
});