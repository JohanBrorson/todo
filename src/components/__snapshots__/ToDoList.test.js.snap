// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToDoList /> renders correctly 1`] = `
<div
  className="center"
>
  <ul />
</div>
`;
